### DISCUSSION TITLE
How to use Calendar Component

### DISCUSSION THREAD

**Discussion by JfrAziz** [kobaltedev/kobalte#290] [General]
Does anybody know how to use Calendar Component?, this component available on `kobalte/core`, but no docs or example how to use it.

Thanks.

**Comment by fabien-ml** [comment]
Hi, the `Calendar` / `DatePicker` are not finished/tested yet. That's why there is no documentation currently.
We have others critical issues that should be fixed before working back on it (in 2024)
